client/state/posts/test/selectors.js

```diff
@@ -33,6 +34,25 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getSitePosts()', () => {
```

Comment:
>I wonder if we should add a util method in createSelector to reset the memo cache.

There is `getSitePosts.memoizedSelector.cache.clear()`, which I should probably be using here in case we start adding more tests ([example usage](https://github.com/Automattic/wp-calypso/blob/341cfd30dd0102d769f39e71c2c989b38bce7487/client/state/posts/test/selectors.js#L232-L234)).